_translator-files/po/es/FAQ.po

```diff
@@ -186,10 +173,8 @@ msgstr "<img src=\"{% include img/es-screenshots/diagram-overview.inc %}\" loadi
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:61
-#, fuzzy
-#| msgid "Jamulus works on the client-server principle. Everybody’s audio is sent to a server, mixed and processed there. Afterwards, the audio is sent back to each client. If a server is made public and registered on a public directory, its information will be broadcast to all clients."
 msgid "Jamulus works on the Client-server principle. Everybody’s audio is sent to a Server, mixed and processed there. Afterwards, the audio is sent back to each Client. If a Server is registered in a Jamulus Directory, the Server's information will be provided to all Clients using that Directory."
```

Comment:
English is wrong here "Client" shouldn't be capitalised.

Reply:
There are capitalisation inconsistencies all over the place.